src/checkers/inference/solver/util/MathUtils.java

```diff
@@ -0,0 +1,36 @@
+package checkers.inference.solver.util;
+
+import javax.lang.model.element.AnnotationMirror;
+
+import checkers.inference.solver.frontend.Lattice;
+
+/**
+ * Methods convert between slot id and Max-SAT id.
+ * 
+ * @author jianchu
+ *
+ */
+public class MathUtils {
```

Comment:
Question: is this `MathUtil` class a general util class for doing Math operations, or it is a specific `Util` class for helping encoding/decoding in the MAX-SAT backend? If it is latter, maybe move this to `maxsat` package would be better?